src/main/java/wooteco/subway/line/LineController.java

```diff
@@ -0,0 +1,51 @@
+package wooteco.subway.line;
+
+import java.net.URI;
+import java.util.List;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class LineController {
+
+    private final LineService lineService;
+
+    public LineController(final LineService lineService) {
+        this.lineService = lineService;
+    }
+
+    @PostMapping("/lines")
+    public ResponseEntity<LineResponse> createLine(@RequestBody LineRequest lineRequest) {
+        LineResponse lineResponse = lineService.createLine(lineRequest);
+        return ResponseEntity.created(URI.create("/lines/" + lineResponse.getId())).body(lineResponse);
```

Comment:
```suggestion
        return ResponseEntity
        .created(URI.create("/lines/" + lineResponse.getId()))
        .body(lineResponse);
```

메서드 체이닝은 내려쓰는것이 가독성과 디버깅 측면에서 모두 좋은 것 같아요.
27 라인에서 에러가 발생한다면 어떤 행위에서 에러가 발생하는지 확인하기 어려울 것 같아요. 🤔 

중간곰 생각은 어떤가요?

Reply:
줄을 바꾸면 확실히 디버깅에 이점이 있겠네요! (어디서 예외가 발생하는지 확실히 보임)
위의 경우에는 한 줄이 길어서 build()를 내리면 가독성도 좋아보입니다.
하지만 아래 같은 경우는 원래 짧은 코드였어서 줄을 바꾸면 아직 어색한 느낌이 드네요.
```java
return ResponseEntity.noContent()
    .build();
```
그렇다고 어떤 부분은 내리고 어떤 부분은 한 줄에 쓰면 일관성이 없으니 줄을 바꾸고 싶습니다.
가독성은 그렇다고 쳐도 디버깅적인 측면 때문에 줄을 내리는 게 더 장점이 크다고 느껴집니다.

이번 미션에서 1~2단계 리팩토링 및 3단계 미션을 페어와 진행하기로 했습니다.
현재 이 부분은 페어와 컨벤션으로 한 줄에 작성하는 것으로 정해서, 추후에 잊지 않고 수정하겠습니다.😄